app/controllers/JoinController.js

```diff
@@ -0,0 +1,56 @@
+var blueprint = require ('@onehilltech/blueprint')
+    request   = require ('superagent')
+  , util      = require ('util')
+  ;
+
+function JoinController () {
+  blueprint.BaseController.call (this);
+}
+
+blueprint.controller (JoinController);
+
+JoinController.prototype.join = function () {
+
+  var self= this;
+
+  return function (req, res) {
+
+      var organization = {
+        "name": req.body.name,
+        "website": req.body.website,
+        "location": req.body.location,
+        "industry": req.body.industry
+      }
+
+      var user = {
+          "email": req.body.email,
+      }
+      .post('localhost:5000/v1/admin/organizations')
+      .send(organization,user)
+      .end(function (err, resp) {
+
+                if (err) {
+                    console.log(err);
+                }
+                else if(doc){
+                    console.log("Sorry, Organization Already Exists!");
+                }
+                else {
+                    result = resp.body.result;
```

Comment:
The successful response body looks as follows:
```
{
  organization: {
    org_id: 'id',
    admin_id: 'id'
  }
}
```
So you can access those values via, req.body.organization.org_id, etc.